File: georocket-server/src/main/java/io/georocket/index/xml/XMLGenericAttributeIndexerFactory.java
package io.georocket.index.xml;

import io.georocket.index.generic.GenericAttributeIndexerFactory;

/**
 * Create instances of {@link XMLGenericAttributeIndexer}
 * @author Michel Kraemer
 */
public class XMLGenericAttributeIndexerFactory extends GenericAttributeIndexerFactory implements XMLIndexerFactory {
  @Override
  public XMLIndexer createIndexer() {
    return new XMLGenericAttributeIndexer();
  }
}

File: georocket-server/src/main/kotlin/io/georocket/index/xml/XMLGenericAttributeIndexerFactory.kt
package io.georocket.index.xml

import io.georocket.index.generic.GenericAttributeIndexerFactory

/**
 * Create instances of [XMLGenericAttributeIndexer]
 * @author Michel Kraemer
 */
class XMLGenericAttributeIndexerFactory : GenericAttributeIndexerFactory(), XMLIndexerFactory {
  override fun createIndexer() = XMLGenericAttributeIndexer()
}
